storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.txt

```diff
@@ -208,12 +221,19 @@ You should check that everything went OK with:
             Seconds_Behind_Master: 13
             ...
 
-Both ``IO`` and ``SQL`` threads need to be running. The ``Seconds_Behind_Master`` means the ``SQL`` currently being executed has a ``current_timestamp`` of 13 seconds ago. It is an estimation of the lag between ``TheMaster`` and ``TheSlave``. Note that at the beginning, a high value could be shown because ``TheSlave`` has to "catch up" with ``TheMaster``.
+Both ``IO`` and ``SQL`` threads need to be running. The
+``Seconds_Behind_Master`` means the ``SQL`` currently being executed has a
+``current_timestamp`` of 13 seconds ago. It is an estimation of the lag between
+``TheMaster`` and ``TheSlave``. Note that at the beginning, a high value could
+be shown because ``TheSlave`` has to "catch up" with ``TheMaster``.
 
```

Comment:
Rewrite the title

Reply:
This is the old description of replication. As you can see it as the txt extension which effectively excludes it from the rendered result: this section won't appear in either html or pdf output.